packages/unmock/src/node/index.ts

```diff
@@ -1,4 +1,4 @@
-import unmock from "unmock-core";
-
 export * from "unmock-core";
+import unmock from "unmock-node";
+export * from "unmock-node";
 export default unmock;
```

Comment:
Do we want `unmock-node` to be the default?

Reply:
Hmm I don't quite understand the question, the default import of `unmock-node` is here set as the default import of `unmock`.

Reply:
My bad, didn't notice the path (thought it's `packages/unmock/index.ts`)